scripts/bfasst/reverse_bit/xray.py

```diff
@@ -95,12 +96,16 @@ def convert_bit_to_fasm(self, bitstream_path, fasm_path):
             proc = subprocess.run(
                 cmd,
                 stdout=fp,
-                stderr=subprocess.STDOUT,
+                stderr=subprocess.PIPE,
```

Comment:
Can you instead send directly to a file?

You can open multiple files at once:  https://stackoverflow.com/questions/4617034/how-can-i-open-multiple-files-using-with-open-in-python